src/olympia/lib/settings_base.py

```diff
@@ -1525,3 +1525,5 @@ def read_only_mode(env):
 SWAGGER_SCHEMA_FILE = path('schema.yml')
 
 SWAGGER_UI_ENABLED = env('SWAGGER_UI_ENABLED', default=False) or TARGET != 'production'
+
+MARKDOWN_EXTENSION_WORKSHOP_URL = 'https://extensionworkshop.com/documentation/develop/create-an-appealing-listing/#make-use-of-markdown'
```

Comment:
this is just hardcoding it in a different place - see the other places `settings.EXTENSION_WORKSHOP_URL` is used in slightly different ways to build urls referencing extensionworkshop - it can either be used directly in the macro, or defined in the views in python (the former is more DRY though).